misago/categories/management/commands/fixcategoriestree.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+
+from misago.acl import version as acl_version
+from misago.categories.models import Category
+
+
+class Command(BaseCommand):
+    """
+    This command rebuilds the thread category tree.
+    It can be useful when the category hierarchy is corrupt due to modifying directly
+    in the database causing MPTT's nested sets to not align correctly.
+    A typical case is when injecting default data into the database from outside misago.
+    """
+    help = 'Rebuilds the thread category tree'
+
+    def handle(self, *args, **options):
+        root = Category.objects.root_category()
+        self.stdout.write("Rebulding category tree_id {}".format(root.tree_id))
```

Comment:
Lets move those messages after the actions they are belonging to, and make them inform about tasks completion, eg. "Categories tree has been rebuild.".

Lets not use `tree_id` in message, those are of little value to user anyway.